File: Data Structures & Algorithms/concatenation-of-array/submission-1.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int n = size(nums);
        vector <int> ans;
        int count = 2;
        while(count){
            for(int i = 0; i < n; i++){
                ans.push_back(nums[i]);
            }
            count--;
        }
        return ans;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq = [0] * len(set(nums))
        arr = list(set(nums))
        ans = []

        for i in range(len(nums)):
            freq[arr.index(nums[i])] += 1

        while(k>0):
            ans.append(arr[freq.index(max(freq))])
            freq[freq.index(max(freq))] = -1
            k -= 1

        return ans